randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -250,11 +253,9 @@ Extra - asset_id: 4
   * Layers: default
   * Pickup 3; Category? Minor
   > Door to Moat
-      All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs
+      Can Lay Any Bombs
   > Door to Attic
-      Morph Ball and Super Missile
+      Super Missile and Can Lay Any Bombs
 
```

Comment:
Same as the above comment, you don't need bombs to get out. Just a super for the block and a way to get back into the tunnel as morph. midair morph(beginner), bombs, or springball should do.